packages/terra-paginator/tests/jest/ControlledPaginator.test.jsx

```diff
@@ -34,6 +35,15 @@ describe('Paginator', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render a Controlled Paginator with visually hidden text', () => {
+    const wrapper = shallowWithIntl(renderWithSelectedPage).dive();
+    wrapper.setState({ breakpoint: 'tiny' });
+    wrapper.update();
+    wrapper.find('PaginatorButton').first().simulate('click', { nativeEvent: { keyCode: null } });
+    expect(wrapper.find('VisuallyHiddenText').at(0).prop('text')).toEqual('Terra.paginator.pageSelected');
```

Comment:
nit: Here and in other instances doesn't `.at(0)' mean the same thing as `.first()`?

Reply:
Used `.first()` to keep consistent: https://github.com/cerner/terra-core/pull/3886/commits/ec98741110d2d3a9ec49885806f8e14497ce5c0d#diff-e05da9d01a4643a4ec7a339a471781e030e36c6b9f8828137fe97345b6934c13R43